scio-core/src/main/scala/com/spotify/scio/util/FunctionsWithSideInput.scala

```diff
@@ -35,6 +35,10 @@ private[scio] object FunctionsWithSideInput {
   def filterFn[T](f: (T, SideInputContext[T]) => Boolean): DoFn[T, T] =
     new SideInputDoFn[T, T] {
       val g = ClosureCleaner.clean(f) // defeat closure
+
+      /*
+       * ProcessContext is required as an argument because it is passed to SideInputContext
+       * */
```

Comment:
Thanks for adding those on every processElement where new syntax can't be applied :heart:

Reply:
Just checking, these outstanding usages shouldn't be impacted by #4496 because we're not calling `c.timestamp()`, right? :)

Reply:
Not sure. In this particular case ProcessContext is exposed with implementations of `trait SideInput[T]`:
https://github.com/spotify/scio/blob/main/scio-core/src/main/scala/com/spotify/scio/values/SideInput.scala#L31

Reply:
I did a quick test with [SideInOutExample](https://github.com/spotify/scio/blob/main/scio-examples/src/main/scala/com/spotify/scio/examples/extra/SideInOutExample.scala) on Dataflow with RunnerV2 and it worked fine, so we should be good! I'll do a quick check on other `c.timestamp()` usages